File: quests/dreadlands/Kunark_Spires.pl
sub EVENT_SPAWN {
   $x = $npc->GetX();
   $y = $npc->GetY();
   quest::set_proximity($x - 120, $x + 120, $y - 120, $y + 120);
}

sub EVENT_ENTER {
   quest::setglobal("spire_dre",1,1,"F"); #when we enter the spires, set global true.
   $spire_dre = undef;
} 

sub EVENT_EXIT {
   quest::delglobal("spire_dre"); #when we leave spires, set global false.
   $spire_dre = undef;
} 

sub EVENT_SIGNAL {
if ($signal == 101) {
$npc->CastSpell(86051,2935); #flavor
}
 }

File: trunk/quests/dreadlands/Kunark_Spires.lua
function event_spawn(e)
	local x = e.self:GetX();
	local y = e.self:GetY();
	
	eq.set_proximity(x - 120, x + 120, y - 120, y + 120);
end

function event_enter(e)
	e.other:SetGlobal("spire_dre", "1", 1, "F");
end

function event_exit(e)
	e.self:SetGlobal("spire_dre", "0", 1, "F");
end

function event_signal(e)
	if (e.signal == 101) then
		e.self:CastSpell(2935, 86051); -- Journey: Luclin
	end
end